tests/meltano/api/controllers/test_sql_controller.py

```diff
@@ -3,27 +3,33 @@
 
```

Comment:
For my own info, what is this file and should this functionality be phased out in the near future? (Not in this PR to be clear, but just want to better understand.)

Reply:
Yea its part of the old `/repos/sync` API path (not sure what that actually does), so would hope this is going to get phased in API 2.0.

Because I renamed the `schedule` fixture, which this test referenced, I had to update it. Then because it hadn't been touch recently, I also had to satisfy flake8/linter. 😮‍💨

Reply:
@aaronsteers fyi - cody is actually removing this as part of https://github.com/meltano/meltano/pull/5965/files#diff-0fc5534da41454b96e1a902d456ef8d91502d8269a306ffb19ba1d6e56560ce4, so this file is getting nuked anyway 🥳